src/test/java/com/redhat/cloud/notifications/events/Lifecycle_BG_ITest.java

```diff
@@ -488,21 +486,6 @@ private void addEventTypeBehaviorGroup(Header identityHeader, String eventTypeId
                 .put("/notifications/eventTypes/{eventTypeId}/behaviorGroups/{behaviorGroupId}")
                 .then()
                 .statusCode(200);
-
-        String responseBody = given()
-                .basePath(API_NOTIFICATIONS_V_1_0)
-                .header(identityHeader)
-                .pathParam("eventTypeId", eventTypeId)
-                .when()
-                .get("/notifications/eventTypes/{eventTypeId}/behaviorGroups")
-                .then()
-                .statusCode(200)
-                .extract().body().asString();
-
-        JsonArray jsonBehaviorGroups = new JsonArray(responseBody);
-        assertEquals(expectedEventTypeBehaviorGroups, jsonBehaviorGroups.size());
-        JsonObject jsonBehaviorGroup = jsonBehaviorGroups.getJsonObject(0);
-        assertEquals(behaviorGroupId, jsonBehaviorGroup.getString("id"));
```

Comment:
These two lines were flawed: there can be more than one behavior group when this method is called.

We didn't need that GET request anyway, this is already done with `checkEventTypeBehaviorGroups`.